adoption-metrics/define_adoption_metrics.md

```diff
@@ -0,0 +1,56 @@
+# What adoption metrics Packit team wants to measure
+
+## Static meaurement (updated periodically)
+
+### Newcomers adoption
+
+- Total amount of newcomers in a given period (monthly/quarterly/half yearly/yearly).
+- How many newcomers (percentage) from a given period already tried some functionality that
+  Packit(-service) provides (COPR, TF, all downstream jobs).
+- How fast newcomers are? How much time (in average) they need from approval to actually
+  do something?
+- How often are different services (COPR, TF, all downstream jobs) used by newcomers?
+  What they use most often and what they don't use at all?
+
+### Active users
+
+Active users are those from whom we have recorded at least one activity in the past period
+and they do not fit into the category of newcomers.
+
+- How often are different services (COPR, TF, all downstream jobs) used by active users?
+  What they use most often and what they use less?
+- How frequently active users use packit services?
+
+### Users retention
+
+- Percentage of previous newcomers who became active Packit users.
+  ((amount of newcomers from previous period becoming an active users
+  / amount of newcomers from previous period) \* 100)
+- What percentage of users are active users? (How many inactive users do we have?)
+
+### Usage of Packit services
+
+- Percentage representation of individual services e.g. (COPR requests / all requests) \* 100
+
+## The following metrics will be displayed as a histograms
```

Comment:
I would like to see it as a heatmap similar to https://www.perfecttrendsystem.com/images/articles/finviz-stock-screener/07-Finviz-stock-heatmap.PNG

Reply:
The heatmap is less clear to me than the histogram, but that's my view. What do others think?

Reply:
as soon as we don't have to spend a month creating that chart, I'm fine with either :D heatmap looks pretty wicked, while histogram is the classic